cli/src/commands/metaedit.rs

```diff
@@ -93,6 +93,12 @@ pub(crate) struct MetaeditArgs {
         value_parser = parse_datetime
     )]
     author_timestamp: Option<Timestamp>,
+
+    /// Update the committer timestamp
+    ///
+    /// This updates the committer date to now, without modifying the committer.
```

Comment:
nit: I think we should document that the committer timestamp will be updated if there are any other changes.

Reply:
Done.